sig-and-wg/wg-knative/kyma-knative-eventing-examples.md

```diff
@@ -0,0 +1,71 @@
+# Examples
```

Comment:
I'm wondering if we should maybe add some description to the examples - like an introductory line about what these examples are about (we can use some information from the other document).

Reply:
Applied the suggestion. Please have a look.